editor/src/components/canvas/canvas-strategies/strategies/grid-rearrange-move-strategy.spec.browser2.tsx

```diff
@@ -412,7 +411,7 @@ export var storyboard = (
         await mouseDragFromPointToPoint(
           editor.renderedDOM.getByTestId(GridCellTestId(EP.fromString('sb/scene/grid/child'))),
           childCenter,
-          offsetPoint(childCenter, windowPoint({ x: 240, y: 240 })),
+          offsetPoint(childCenter, windowPoint({ x: 1200, y: 1200 })),
```

Comment:
what was the reason for all these coordinate changes? I thought there are no functional changes, we achieve the same outcome but using two strategies